cfme/tests/webui/test_general_ui.py

```diff
@@ -461,19 +461,64 @@ def test_configure_icons_roles_by_server():
 @pytest.mark.tier(0)
 def test_create_filter_with_multiple_conditions():
     """
-        Polarion:
-        assignee: pvala
-        casecomponent: WebUI
-        caseimportance: medium
-        initialEstimate: 1/5h
-        setup:
-            1. Navigate to Compute > Infrastructure > Providers.
-            2. Click on Advanced Search Filter.
-        testSteps:
-            1. Create an expression with multiple types of condition. Eg: arg_1 AND arg_2 OR arg_3
-        expectedResults:
-            2. Expression must be created successfully.
+    Polarion:
+    assignee: pvala
+    casecomponent: WebUI
+    caseimportance: medium
+    initialEstimate: 1/5h
+    setup:
+        1. Navigate to Compute > Infrastructure > Providers.
+        2. Click on Advanced Search Filter.
+    testSteps:
+        1. Create an expression with multiple types of condition. Eg: arg_1 AND arg_2 OR arg_3
+    expectedResults:
+        2. Expression must be created successfully.
```

Comment:
Here, the expectedResults number should match the testSteps number.

Reply:
Yes, it definitely should. :roll_eyes: